.github/workflows/code_quality.yml

```diff
@@ -0,0 +1,33 @@
+name: Code Quality (rustfmt and clippy)
```

Comment:
```suggestion
name: Code Quality
```

The UI shows the job names, so there's no need to repeat them here.

Reply:
done